include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1714,6 +1714,33 @@ int bt_le_ext_adv_delete(struct bt_le_ext_adv *adv);
  */
 uint8_t bt_le_ext_adv_get_index(struct bt_le_ext_adv *adv);
 
+/** Advertising states */
+enum bt_le_ext_adv_state {
+	/** No state */
+	BT_LE_EXT_ADV_STATE_NONE,
+
+	/** The advertising set has been created but not started */
+	BT_LE_EXT_ADV_STATE_CREATED,
```

Comment:
The advertiser will go into this state after the user stops it, right? I think "stopped" may be a better name for this state rather than "created". What do you think?

Reply:
Sure, that makes sense. Could rename these _ENABLED and _DISABLED

Reply:
Prefer ENABLED and DISABLED. STOPPED sounds like more like it has actively been stopped, which isn't always the case.